assets/src/edit-story/app/media/media3p/api/test/useMedia3pApi.js

```diff
@@ -194,6 +204,83 @@ describe('useMedia3pApi', () => {
     });
   });
 
+  it('should call listMedia with searchTerm', async () => {
+    const wrapper = (params) => (
+      <Media3pApiProvider>{params.children}</Media3pApiProvider>
+    );
+    const { result } = renderHook(() => useMedia3pApi(), { wrapper });
+
+    await result.current.actions.listMedia({
+      provider: 'unsplash',
+      searchTerm: 'cat',
+    });
+
+    expect(apiFetcherMock.listMedia).toHaveBeenCalledWith({
+      filter: 'provider:unsplash cat',
+      orderBy: undefined,
+      pageSize: 20,
+      pageToken: undefined,
+    });
+  });
+
+  it('should call listMedia with category and ignored search term', async () => {
```

Comment:
It feels like the caller of listMedia() should explicitly drop the search term.

Reply:
Done.